### PR TITLE
demulshooter: Update to version 12

### PR DESCRIPTION
**Pull request by RavenMacDaddy**
Manual update to version 12; the `autoupdate` section would need `$cleanVersion` at the end, while more likely than not, `$version` will continue to work going forward, since everything points to the next version being called 12.1, with the URL following that.

- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by RavenMacDaddy** [comment]
CI?

> Thanks!
> 
> Should be good to merge after fixing CI.

**Comment by Calinou** [comment]
> This right? My interpretation is that you're waiting for other code to merge according to some sort of schedule.

Yes, look at the **Checks** tab of the pull request as 1 check is currently failing: https://github.com/Calinou/scoop-games/pull/1109#issuecomment-2002370564

**Comment by RavenMacDaddy** [comment]
Since the whole thing is about a manual update while waiting for release 12.1, you're free to either close or merge it as is — your call. :)

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**
